src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -74,7 +78,19 @@ public function __construct(
 
     public function isHandling(LogRecord $record): bool
     {
-        return $this->updateLevel() && parent::isHandling($record);
+        return
+            $this->updateLevel()
+            && parent::isHandling($record)
+            && (!$this->interactiveOnly || ($this->input && $this->input->isInteractive()));
+    }
+
+    public function getBubble(): bool
+    {
+        if ($this->isInteractiveOnlyEnabled()) {
```

Comment:
The private function `isInteractiveOnlyEnabled` is only called once, it should be inlined.
```suggestion
        if ($this->interactiveOnly && $this->input?->isInteractive()) {
```